pkg/rtc/participant.go

```diff
@@ -707,6 +707,7 @@ func (p *ParticipantImpl) GetConnectionQuality() livekit.ConnectionQuality {
 		if subTrack.IsMuted() {
 			continue
 		}
+		// LK-TODO: maybe this should check CurrentSpatialLayer as target may not have been achieved
```

Comment:
@davidzhao more of a question for you here. Should we look at current spatial layer for connection quality stuff?